data/tools/unit_tree/html_output.py

```diff
@@ -1143,30 +1163,30 @@ def clean_uval(name):
         write('</td></tr>\n')
 
         attributes = [
-            ("cost",       _("Cost: ")),
-            ("hitpoints",  _("HP: ")),
-            ("movement",   _("Moves: ")),
-            ("vision",     _("Vision: ")),
-            ("jamming",    _("Jamming: ")),
-            ("experience", _("XP: ")),
-            ("level",      _("Level: ")),
-            ("alignment",  _("Alignment: ")),
-            ("id",         "Id: ")
+            ("cost",       _w_editor("Cost: ")),
```

Comment:
Removing the trailing space finds a lot of these strings in `wesnoth-help`, which makes sense to me as it's where the in-game help page should come from.

That may also snowball into working out which other strings in wesnoth-help shouldn't have trailing spaces, will try and find a simpler way to unblock #7627.